pkgs/applications/display-managers/sddm/sddm-catppuccin.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchFromGitHub, flavor ? "mocha" }:
+stdenv.mkDerivation {
+  pname = "sddm-catppuccin-theme";
+  version = "0.0.0b0";
+
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/share/sddm/themes
+    mv src/catppuccin-${flavor} $out/share/sddm/themes/catppuccin-${flavor}
+  '';
+
+  src = fetchFromGitHub {
```

Comment:
src should be ordered after version

Reply:
resolved in [517edfb](https://github.com/NixOS/nixpkgs/pull/240990/commits/517edfbaeaad7c040ffa212334e4584816f2a2d1)